packages/components/src/input/src/components/osds-input/osds-input.tsx

```diff
@@ -36,49 +37,45 @@ export class OsdsInput implements OdsInputAttribute, OdsInputEvent, OdsInputMeth
 
   @Prop() ariaLabel: HTMLElement['ariaLabel'] = DEFAULT_ATTRIBUTE.ariaLabel;
 
-  @Prop() ariaLabelledby?: string = DEFAULT_ATTRIBUTE.ariaLabelledby;
+  @Prop() ariaLabelledby?: string;
 
-  @Prop({ reflect: true }) clearable?: boolean = DEFAULT_ATTRIBUTE.clearable;
-
-  @Prop({ reflect: true }) color?: ODS_THEME_COLOR_INTENT = DEFAULT_ATTRIBUTE.color;
+  @Prop({ reflect: true }) clearable?: boolean;
 
   @Prop({ reflect: true }) defaultValue: OdsInputValue = DEFAULT_ATTRIBUTE.defaultValue;
 
   @Prop({ reflect: true }) disabled: boolean = DEFAULT_ATTRIBUTE.disabled;
 
   @Prop({ reflect: true }) error: boolean = DEFAULT_ATTRIBUTE.error;
 
-  @Prop({ reflect: true }) forbiddenValues?: OdsInputValue[] = DEFAULT_ATTRIBUTE.forbiddenValues;
+  @Prop({ reflect: true }) forbiddenValues?: OdsInputValue[];
 
-  @Prop({ reflect: true }) icon?: ODS_ICON_NAME = DEFAULT_ATTRIBUTE.icon;
+  @Prop({ reflect: true }) icon?: ODS_ICON_NAME;
 
   @Prop({ reflect: true }) inline?: boolean = DEFAULT_ATTRIBUTE.inline;
 
-  @Prop({ reflect: true }) label?: string = DEFAULT_ATTRIBUTE.label;
+  @Prop({ reflect: true }) label?: string;
 
-  @Prop({ reflect: true }) loading?: boolean = DEFAULT_ATTRIBUTE.loading;
+  @Prop({ reflect: true }) loading?: boolean;
 
-  @Prop({ mutable: true, reflect: true }) masked?: boolean = DEFAULT_ATTRIBUTE.masked;
+  @Prop({ mutable: true, reflect: true }) masked?: boolean;
 
-  @Prop({ reflect: true }) max?: number = DEFAULT_ATTRIBUTE.max;
+  @Prop({ reflect: true }) max?: number;
 
-  @Prop({ reflect: true }) min?: number = DEFAULT_ATTRIBUTE.min;
+  @Prop({ reflect: true }) min?: number;
 
   @Prop({ reflect: true }) name: string = DEFAULT_ATTRIBUTE.name;
 
-  @Prop({ reflect: true }) placeholder?: string = DEFAULT_ATTRIBUTE.placeholder;
-
-  @Prop({ reflect: true }) prefixValue = DEFAULT_ATTRIBUTE.prefixValue;
+  @Prop({ reflect: true }) placeholder?: string;
 
-  @Prop({ reflect: true }) readOnly?: boolean = DEFAULT_ATTRIBUTE.readOnly;
+  @Prop({ reflect: true }) prefixValue? : string;
 
-  @Prop({ reflect: true }) required?: boolean = DEFAULT_ATTRIBUTE.required;
+  @Prop({ reflect: true }) readOnly?: boolean;
 
-  @Prop({ reflect: true }) size?: ODS_COMMON_FIELD_SIZE = DEFAULT_ATTRIBUTE.size;
+  @Prop({ reflect: true }) required?: boolean;
 
-  @Prop({ reflect: true }) step?: number = DEFAULT_ATTRIBUTE.step;
+  @Prop({ reflect: true }) step?: number;
 
-  @Prop({ reflect: true }) type?: ODS_COMMON_INPUT_TYPE = DEFAULT_ATTRIBUTE.type;
+  @Prop({ reflect: true }) type?: ODS_INPUT_TYPE = DEFAULT_ATTRIBUTE.type;
```

Comment:
BTW you can remove type when using DEFAULT_ATTR, Stencil v4.x+ now infer correctly

Reply:
I have an error on the component.d.ts without the type. The import is missing the the components.d.ts